### PR TITLE
Add Send an SMS / Outbound Voice example

### LINKED ISSUE #11: SMS > Sending an SMS
See gist:

https://gist.github.com/adambutler/fdfeba152303388adc3fbc7c99272123

### PR DESCRIPTION
**Pull request by adambutler**
Fixes #11 & #12

### CODE REVIEW

**Comment by sammachin** [comment]
the requirements.txt looks a little strange, there's A LOT of stuff in there for what should be really simple code, the only things I can think we should need for building blocks are the nexmo libraryr (with greater than or equal to min version) and perhaps flask for inbound webhooks?

**Comment by adambutler** [comment]
@sammachin I agree and thought that might be the case when I modified the file to add the dotenv dependency.

This, however, should be considered separately from this PR and I'm probably not the right person to do that. Any further thoughts on the changes in this PR?

**Comment by adambutler** [comment]
Update: Since this hasn't been reviewed yet and they share the same `dotenv` dependency I've expanded this PR to also include the outbound voice building block.

File: sms/send-an-sms.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.environ.get('NEXMO_API_KEY')
```
Comment (judy2k) [inline sms/send-an-sms.py]: I would usually use `os.getenv` but for some reason that's just personal preference. Just an FYI - happy for you to leave.

File: voice/make-an-outbound-call.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+APPLICATION_ID = os.environ.get('APPLICATION_ID')
+APPLICATION_PRIVATE_KEY_PATH = os.environ.get('APPLICATION_PRIVATE_KEY_PATH')
+TO_NUMBER = os.environ.get('TO_NUMBER')
+NEXMO_NUMBER = os.environ.get('NEXMO_NUMBER')
+APPLICATION_PRIVATE_KEY = open('private.key','r').read()
```
Comment (judy2k) [inline voice/make-an-outbound-call.py]: No need to read the key in - `NexmoClient` will automatically detect a key vs a path, and read in the file if necessary.

Comment (judy2k) [inline voice/make-an-outbound-call.py]: Could you upgrade `nexmo` to 2.0.0?

File: voice/make-an-outbound-call.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import nexmo
+
+client = nexmo.Client(
+    application_id=APPLICATION_ID,
+    private_key=APPLICATION_PRIVATE_KEY
```
Comment (judy2k) [inline voice/make-an-outbound-call.py]: I would normally put a trailing comma here, but I try not to be dogmatic about it :)

**Comment by judy2k** [comment]
The requirements.txt file is listing a lot of transient dependencies - as Sam says, it should pretty much consist of `nexmo=~2.0.0` and `cryptography` ... but get this merged and we'll sort that out later - this won't hurt, and if anything, it has some benefits.

**Pull request merged**